src/views/Dashboard.vue

```diff
@@ -60,7 +60,7 @@
 
             <section>
               <header class="aside-header mb-3">
-                <h2 data-test-id="dashboard-addresses-subtitle">Office Addresses</h2>
+                <h2 data-test-id="dashboard-addresses-subtitle">{{addressHeader}}</h2>
```

Comment:
I'm thinking line 84 (`:disabled`) should also check the hasBlocker getter. Or is there something else that disables this button if there's a blocker filing?

Reply:
I think value passed to the showGrayedOut prop already takes care of this 
![image](https://user-images.githubusercontent.com/8199923/164814632-50b9f3d0-2fcd-46f7-8dcf-4ea31a89cefb.png)

Reply:
Thanks for checking. I see now that `isAllowed(AllowableActions.FILE_ADDRESS_CHANGE)` checks for hasBlocker.